stable/aws-for-fluent-bit/values.yaml

```diff
@@ -20,7 +20,7 @@ service:
   #   HTTP_PORT    2020
   parsersFiles:
     - /fluent-bit/parsers/parsers.conf
-  # additionalParsers: |
+  # extraParsers: |
```

Comment:
hmmm it seems here we are changing the actual values in the template which may break folks?